src/test/kotlin/com/github/imflog/schema/registry/config/ConfigTaskActionTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.github.imflog.schema.registry.config
+
+import io.confluent.kafka.schemaregistry.client.MockSchemaRegistryClient
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+
+class ConfigTaskActionTest {
+
+    @Test
+    fun `Should set subject config`() {
+        // given
+        val registryClient = MockSchemaRegistryClient()
+
+        val subjects = listOf(
+            Pair(
+                "test",
+                "FULL_TRANSITIVE"
+            )
+        )
+
+        // when
+        val errorCount = ConfigTaskAction(
+            registryClient,
+            subjects
+        ).run()
+
+        // then
+        Assertions.assertThat(errorCount).isEqualTo(0)
```

Comment:
If we use the AvroCompatilibityLevels, we should test that we call the mockClient with the right parameters (using mockk verify).

Reply:
Good idea, but as above, not possible.